config/locales/in_person_proofing/en.yml

```diff
@@ -57,16 +57,11 @@ en:
           ID in person.'
     form:
       address:
-        address1: Address
```

Comment:
do we need to remove equivalent fr, es translations?

Reply:
Whoops, yes. Thankfully the build caught that too.

Reply:
Updated in 2fd41a0.